### DISCUSSION TITLE
change the size of radio buttons in likert trials

### DISCUSSION THREAD

**Discussion by sonhkim** [jspsych/jsPsych#1087] [Q&A] [answered]
Hi jsPsychers, 

I'm pretty new to css and javascript in general. Is it possible to change the size of radio buttons in survey-likert type trials? I want to make them bigger. I've been trying to tweak some lines in the following block in jspsych-suvey-likert.js but haven't succeeded. Any suggestions? 

var html = "";
    // inject CSS for trial
    html += '<style id="jspsych-survey-likert-css">';
    html += ".jspsych-survey-likert-statement { display:block; font-size: 16px; padding-top: 40px; margin-bottom:10px; }"+
      ".jspsych-survey-likert-opts { list-style:none; width:"+w+"; margin:auto; padding:0 0 35px; display:block; font-size: 14px; line-height:1.1em; }"+
      ".jspsych-survey-likert-opt-label { line-height: 1.1em; color: #444; }"+
      ".jspsych-survey-likert-opts:before { content: ''; position:relative; top:11px; /*left:9.5%;*/ display:block; background-color:#efefef; height:4px; width:100%; }"+
      ".jspsych-survey-likert-opts:last-of-type { border-bottom: 0; }"+
      ".jspsych-survey-likert-opts li { display:inline-block; /*width:19%;*/ text-align:center; vertical-align: top; }"+
      ".jspsych-survey-likert-opts li input[type=radio] { display:block; position:relative; top:0; left:50%; margin-left:-6px; }"
    html += '</style>';

**Comment by becky-gilbert** [comment, marked as answer]
Hi @sonhkim! The comments on this [stack overflow question](https://stackoverflow.com/questions/4920281/how-to-change-the-size-of-the-radio-button-using-css) mostly suggest setting the height and width of the input[type="radio"] elements. And if you add `vertical-align: middle` then the larger radio buttons will be centered with the option text, rather than sticking out above it. 

If you want to edit the survey-likert plugin, then you can add this to the `html` string:
```javascript
'input[type="text"] {height: 35px; width: 35px; vertical-align: middle;}'
```
Or you could add some `<style>` tags to the `<head>` section of your HTML file, like this:
```html
<head>
  <script src="jspsych-6.1.0/jspsych.js"></script>
  ...
  <style>
    input[type="radio"] {height: 35px; width: 35px; vertical-align: middle;}
  </style>
</head>
```

**Reply by sonhkim** [reply]
Thanks, it worked!
